core/src/main/scala/ai/forestflow/utils/SourceStorageProtocols.scala

```diff
@@ -17,23 +17,35 @@ import java.net.URI
 import java.nio.file.Paths
 
 import ai.forestflow.domain.{Contract, FQRV}
+import akka.NotUsed
+import akka.actor.ActorSystem
+import akka.stream.alpakka.s3.{ObjectMetadata, S3Attributes, S3Ext, S3Settings, scaladsl}
+import akka.stream.{ActorMaterializer, IOResult}
+import akka.stream.scaladsl.{FileIO, Source}
+import akka.util.{ByteString, Timeout}
 import com.typesafe.scalalogging.StrictLogging
 import org.apache.commons.io.FileUtils
 import org.eclipse.jgit.api.CloneCommand
 import org.eclipse.jgit.storage.file.FileRepositoryBuilder
 import org.eclipse.jgit.transport.{CredentialItem, CredentialsProvider, URIish}
 
+import scala.concurrent.{Await, ExecutionContext, Future}
 import scala.language.implicitConversions
+import scala.util.{Failure, Success}
+import scala.concurrent.duration._
+
 
 /**
   * The different protocols we can understand and support
   */
-object SourceStorageProtocols extends StrictLogging {
+object  SourceStorageProtocols extends StrictLogging {
```

Comment:
extra space between "object" and SourceStorageProtocols.
Please revert to old/correct formatting.